README.md

```diff
@@ -13,79 +13,228 @@
 buildpack_main!(HelloWorldBuildpack);
+
+```
+
+### Packaging the Buildpack
+Now that our buildpack is written, it's time to package it, so that it can be run. If you followed the steps to setup
+your development environment, you have access to the `libcnb` Cargo command that will handle packaging for you.
+
+In your project directory, run `cargo libcnb package` to start packaging:
+
+```shell
+$ cargo libcnb package
+INFO - Reading buildpack metadata...
+INFO - Found valid buildpack with id "libcnb-examples/my-buildpack" @ 0.1.0!
+INFO - Building buildpack binary (x86_64-unknown-linux-musl)...
+   Compiling syn v1.0.82
+   Compiling bit-vec v0.6.3
+   Compiling regex-syntax v0.6.25
+   # Omitting further compilation output...
+    Finished dev [unoptimized + debuginfo] target(s) in 19.71s
+INFO - Writing buildpack tarball...
+INFO - Successfully wrote buildpack tarball target/libcnb-examples_my-buildpack_0.1.0_dev.tar.gz (6.5M)
+INFO - Packaging successfully finished!
+INFO - Hint: To test your buildpack locally with pack, run: pack build my-image -b target/libcnb-examples_my-buildpack_0.1.0_dev.tar.gz --path /path/to/application
 ```
+
+If you get errors with hints about how to install required tools to cross-compile from your host platform to the
+target platform, follow them and try again. These differ from platform to platform, that's why they're not part of this
+quick start guide.
+
+### Running the Buildpack
+You might have seen in the hint in the output of the packaging step that we're now ready to run our buildpack locally 
+with `pack`. Before we can do this, we need an application to build. Since our buildpack does not interact with the
+application code at all, we just create an empty directory and use that as our application:
+
+```shell
+$ mkdir bogus-app
+$ pack build my-image -b target/libcnb-examples_my-buildpack_0.1.0_dev.tar.gz --path bogus-app
+20: Pulling from heroku/buildpacks
+Digest: sha256:04e8ea7a1f482f289d432d9518edcfaaf9f3a10432cd1b624e58225f22e7c416
+Status: Image is up to date for heroku/buildpacks:20
+20: Pulling from heroku/pack
+Digest: sha256:21ea4b85bc937b47e017bf43136a5295ec08e093e3c210b20e69e9c1b0f2bd57
+Status: Image is up to date for heroku/pack:20
+===> DETECTING
+libcnb-examples/my-buildpack 0.1.0
+===> ANALYZING
+Skipping buildpack layer analysis
+===> BUILDING
+Hello World!
+Build runs on stack heroku-20!
+===> EXPORTING
+Reusing 1/1 app layer(s)
+Reusing layer 'launcher'
+Reusing layer 'config'
+Reusing layer 'process-types'
+Adding label 'io.buildpacks.lifecycle.metadata'
+Adding label 'io.buildpacks.build.metadata'
+Adding label 'io.buildpacks.project.metadata'
+Setting default process type 'web'
+Saving my-image...
+*** Images (d4f67a828236):
+      my-image
+Successfully built image my-image
+```
+
+### Running the image
+Just run the crated Docker image as you normally would. If all went well, you should see our "Hello World!" message in
+your terminal:
```

Comment:
"Just" can sometimes read a bit negative ("oh you just have to do X, it's not that hard"). The usage here isn't too bad, but perhaps something like this would be gentler?

```suggestion
The newly created Docker image can be run in the same way as you would a Docker image created via `docker build`.
If all went well, you should see our "Hello World!" message in your terminal:
```